### PR TITLE
Add Custom Loader

### PR DESCRIPTION
**Pull request by jyash97**
Batteries Commit - Dev latest

### CODE REVIEW

File: src/components/Loader/Loader.js [hunk 1]
```diff
@@ -11,9 +10,49 @@ const styles = css`
...
+		}
+	}
+	transform-origin: 75px 75px;
+	animation: loading linear 1s infinite;
+`;
+
 const Loader = () => (
 	<div className={styles}>
-		<Spin size="large" />
+		<svg
```
[the remark this answers is not in the document]
Comment (jyash97) [reply src/components/Loader/Loader.js]: Than how we would be applying the css on that, By passing Props ?
And what's the advantage to use img element ?

[the remark this answers is not in the document]
Comment (metagrover) [reply src/components/Loader/Loader.js]: We can apply styles and refer the dom unit in the css to rotate accordingly. I'm not super certain but I've seen browser painting taking a more time/processing with this method.

[the remark this answers is not in the document]
Comment (jyash97) [reply src/components/Loader/Loader.js]: Okay we can change it to `img`

**Comment by metagrover** [comment]
@siddharthlatest Is this good to go?

**Pull request merged**